bin/pint

```diff
@@ -150,22 +151,94 @@ if command_args['--region']:
 
 try:
     if command_args['images']:
-        data = ifsrequest.get_image_data(
-            framework,
-            image_state,
-            output_format,
-            region,
-            command_args['--filter'])
-        if '<images/>' in data or '"images": []' in data:
-            data = 'No information available. Please check your filter'
-        print(data)
+        info = False
+        no_info = ''
+        data_json = {}
+        data_xml = None
```

Comment:
I don't think any flags are needed. We take the users input and provide the information for that input. How would we even guess whether a region name is wrong or an image query is bogus?

Reply:
The flags were initially to check if no info and print a message, but that is not the case anymore